src/settings.ts

```diff
@@ -34,7 +34,7 @@ export const CREATE_EDIT_URL = "/admin";
 export const SPARKLEVERSITY_URL = "https://sparklever.se/sparkleversity";
 export const SPARKLEVERSE_COMMUNITY_URL =
   "https://www.facebook.com/groups/sparkleverse/";
-export const TICKET_URL = "https://sparklever.se/glowforth/";
+export const TICKET_URL = "https://www.eventbrite.co.uk/e/129232464781";
```

Comment:
[OOS, future] We really need to get around to making this a param/env var/similar

Reply:
Agreed 👍